index.html

```diff
@@ -2435,37 +2435,30 @@ <h2>Cryptographic Suites</h2>
 verification of the data.
         </li>
         <li>
-          The specification MUST define a <dfn>suite selection algorithm</dfn> that accepts a [=JSON
-          object=] |proof| and uses the properties described in [[[#proofs]]] to return either
-          failure or a [=VC cryptographic suite=] [[Infra]] [=struct=]. This algorithm SHOULD be
-          registered in [[[VC-SPECS]]]. A
-          <dfn class="export">VC cryptographic suite</dfn> [=struct=] has the following
-          [=struct/items=].
-          <dl data-dfn-for="cryptographic suite">
-            <dt><dfn>transform</dfn></dt>
-            <dd>A [=transformation algorithm=] that takes a [=JSON object=] as input and returns a
-            [=JSON object=].</dd>
-            <dt><dfn>hash</dfn></dt>
-            <dd>A [=hashing algorithm=] that takes a [=JSON object=] as input and returns a [=byte
-            sequence=].</dd>
-            <dt><dfn>proof creation options</dfn></dt>
+The specification MUST define a <dfn class="export"
+data-lt="cryptosuite instantiation algorithm">data integrity cryptographic
+suite instantiation algorithm</dfn> that accepts a set of options
+([=map=] |options|) and returns a [=cryptosuite instance=]
+([=struct=] |cryptosuite|). This algorithm SHOULD be listed in [[[VC-SPECS]]].
+A <dfn class="export" data-lt="cryptosuite instance">data integrity
+cryptographic suite instance</dfn> [=struct=] has the following
+[=struct/items=].
+          <dl data-dfn-for="cryptosuite instance">
+            <dt><dfn>createProof</dfn></dt>
             <dd>
-              <p>A [=struct=] type that needs to be supplied to the [=cryptographic suite/serialize
-              proof=] operation.</p>
-              <p class="note">To sign data, cryptographic suites need to be able to use a private
-              key. This private key will often be implied by a field in the [=proof creation
-              options=], but it is often a better design to avoid passing the key directly. Passing
-              a key identifier or an asynchronous signing algorithm allows implementations to take
-              advantage of hardware security modules (HSMs) to get better security.</p>
+An algorithm that takes an [=unsecured data document=] ([=map=]
+|unsecuredDocument|) and proof options ([=map=] |options|) as input, and
+produces a [=data integrity proof=] ([=map=]) or an error.
+            </dd>
+            <dt><dfn>verifyProof</dfn></dt>
+            <dd>
+An algorithm that takes a [=secured data document=] ([=map=]
+|securedDocument|) as input, and produces a verification result
+([=map=]) or an error. The verification result MUST contain a
+verification status ([=boolean=] |verified|) value and an [=unsecured data
+document=] ([=map=] |verifiedDocument|), and MAY contain information
```

Comment:
I'd make "verifiedDocument" into a `<dfn>`. If there's more than one defined key, I'd then use a `<dl>` to organize them, but you may only have the 1 after handling my previous comment.

```suggestion
<dfn data-dfn-for="cryptosuite instance/verifyProof">verifiedDocument</dfn>
[=map/key=] whose [=map/value=] is an [=unsecured data
document=]. Other keys MAY contain information
```

Reply:
Fixed in 1f7ee96701cccc2c1e00992ef61bc397d3bb8388.